### ISSUE TITLE
Change MessageFormatter's function signature

### ISSUE THREAD

**Issue by dkzlv** [kaisermann/svelte-i18n#109]
**Is your feature request related to a problem? Please describe.**
Currently there's no simple way to use the embedded `$format` (`_`) in TypeScript, since it returns a function, that returns `string | unknown`. You cannot pass values as props with `string` type, you cannot use it in most of the HTML attributes (like `placeholder`), etc.

**Describe the solution you'd like**
Change signature for `MessageFormatter`'s function to `string` instead of `string | unknown`. Or `any` if `unknown` is there for a very valid reason (not sure, the documentation [here](https://github.com/kaisermann/svelte-i18n/blob/main/docs/Formatting.md#format-or-_-or-t) states that it always returns string).

**Describe alternatives you've considered**
I've wrapped it myself in my project.
```
import { _ as $format } from 'svelte-i18n';
import { MessageObject } from 'svelte-i18n/types/runtime/types';
import { Readable } from 'svelte/store';

export const _ = $format as Readable<
  (id: string | MessageObject, options?: MessageObject) => string
>;

```
Not very clean or beautiful, tbh.

**How important is this feature to you?**
Quite!

**Comment by kaisermann** [comment]
Hey 👋  fixed in 3.2.7! Since 3.2.0, the format method can also return an object or array, but setting it to return `unknown` was indeed a mistake.

**Comment by dkzlv** [comment]
@kaisermann Holy shit you're fast. Thanks! ❤️
